driverloader/driverloader_linux.go

```diff
@@ -2,38 +2,43 @@ package driverloader
 
 import (
 	"fmt"
+	"os"
 	"strings"
 
 	"github.com/hyperhq/runv/hypervisor"
 	"github.com/hyperhq/runv/hypervisor/libvirt"
 	"github.com/hyperhq/runv/hypervisor/qemu"
-	"github.com/hyperhq/runv/hypervisor/vbox"
 	"github.com/hyperhq/runv/hypervisor/xen"
 )
 
 func Probe(driver string) (hypervisor.HypervisorDriver, error) {
 	switch strings.ToLower(driver) {
-	case "vbox":
-		vd := vbox.InitDriver()
-		if vd != nil {
-			fmt.Printf("Vbox Driver Loaded.\n")
-			return vd, nil
+	case "libvirt":
+		ld := libvirt.InitDriver()
+		if ld != nil {
+			fmt.Printf("Libvirt Driver Loaded.\n")
+			return ld, nil
+		}
+	case "kvm", "qemu-kvm":
+		if _, err := os.Stat("/dev/kvm"); os.IsNotExist(err) {
+			return nil, fmt.Errorf("Driver %s is unavailable\n", driver)
 		}
-	case "xen":
+		qd := qemu.InitDriver()
+		if qd != nil {
+			fmt.Printf("%s Driver Loaded\n", driver)
+			return qd, nil
+		}
+	case "xen", "":
```

Comment:
only if the xen driver is available. otherwise it will use qemu.
and qemu driver uses kvm if kvm is available.